### DISCUSSION TITLE
Welcome to solutions-architecture-patterns Discussions!

### DISCUSSION THREAD

**Discussion by chanakaudaya** [chanakaudaya/solutions-architecture-patterns#16] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ttrit** [comment]
Hi @chanakaudaya, recently I found this repo is amazing and want to share it with others. So can I translate this to Vietnamese and create a fork of this repo?
